### DISCUSSION TITLE
求教大佬，如何批量识别

### DISCUSSION THREAD

**Discussion by xiaxuesis** [RapidAI/RapidTable#2] [Q&A] [answered]
我通过magic-pdf将pdf文件中的表格批量导出为jpg了，请问通过大佬的工具，怎么批量把导出文件夹里面的jpg识别并写为html或者markdown文件啊？

**Comment by SWHL** [comment, marked as answer]
刚看到，自己写一个for循环，逐个识别一下就可以了吧
